core/src/main/scala/slickeffect/DBIOSync.scala

```diff
@@ -0,0 +1,34 @@
+package slickeffect
+
+import slick.dbio.DBIO
+
+import scala.util.{Failure, Success}
+import cats.effect.kernel.Sync
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext
+
+private[slickeffect] class DBIOSync(implicit ec: ExecutionContext) extends Sync[DBIO] {
```

Comment:
🤔 Are we able to do *anything* else? `Temporal` maybe?

Reply:
`Temporal` implies `Concurrent` (can't do because we can't spawn a fiber). We have `Clock` because `Sync` extends that :)